cpp/core/utils/TaskContext.cc

```diff
@@ -61,12 +64,13 @@ void bindToTask(std::shared_ptr<void> object) {
   std::lock_guard<std::mutex> guard(fallbackStorageMutex);
   std::cout << "Binding a shared object to fallback storage. This should only happen on sub-thread of a Spark task. "
             << std::endl;
+  GLUTEN_CHECK(false, "Force throw");
```

Comment:
Does this mean we no longer need the fallback storage at all?

Reply:
It's an attempt, I have passed all tpc queries with this check.